rviz2/package.xml

```diff
@@ -40,12 +40,9 @@
   <depend>rviz_ogre_vendor</depend>
 
   <!-- TODO(jacobperron): Replace with ament_lint_common when ament_copyright is working -->
```

Comment:
This TODO comment can be removed now.

Reply:
https://github.com/ros2/rviz/pull/1231/commits/b2d4d1b09772da1468065aca1b8320a9813c6d98